sci-mathematics/jags/jags-4.0.0.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2013 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+inherit autotools-utils toolchain-funcs
+
+MYP="JAGS-${PV}"
+
+DESCRIPTION="Just Another Gibbs Sampler for Bayesian MCMC simulation"
+HOMEPAGE="http://mcmc-jags.sourceforge.net/"
+SRC_URI="mirror://sourceforge/project/mcmc-jags/JAGS/4.x/Source/${MYP}.tar.gz"
+LICENSE="GPL-2"
+IUSE="doc"
+
+SLOT="0"
+KEYWORDS="amd64 x86 ~amd64-linux ~x86-linux"
```

Comment:
Please do not add new versions with stable keywords. You can use `ekeyword ~all jags-4.0.0.ebuild` to automatically lower them.